activedirectory/azure-ad-integration-spring-boot-autoconfigure/README.md

```diff
@@ -0,0 +1,40 @@
+## Overview
+This package provides a Spring Security filter to validate the Jwt token from Azure AD. The Jwt token is also used to acquire a On-Behalf-Of token for Azure AD Graph API so that authenticated user's membership information is available for authorization of access of API resources. 
```

Comment:
Should insert a paragraph that describes the scenario that this enables, ie using Azure AD to implement authN/Z for spring-boot based Rest APIs.